src/Symfony/Component/Console/Style/SymfonyStyle.php

```diff
@@ -369,6 +374,29 @@ private function getProgressBar(): ProgressBar
             ?? throw new RuntimeException('The ProgressBar is not started.');
     }
 
+    public function createTree(TreeNode|array|\Iterator|null $node = null): Tree
```

Comment:
Not tested ? Is it needed for something ? Like createTable ?

Reply:
Exactly... I'll rework it if i find an elegant solution for the generator. It will be much easier with the cycle detection made on render.